m1: Fundamentos/Bloco 7 - Introdução à ES6 e Testes Unitários/01 - JavaScript ES6 - let, const, arrow functions e template literals/parte2.js

```diff
@@ -0,0 +1,45 @@
+// 1. Crie uma função que receba um número e retorne seu fatorial.
+const fatorial = (num) => {
```

Comment:
`(n > 1) ? n * fatorial(n - 1) : n;` resolveria esse problema em uma linha. For gasta mais memoria.

Reply:
Sim, depois no gabarito que dava pra fazer assim. Porém só copiar de lá por ser mais limpo, não faria com que eu me sentisse bem depois.